app/static/scripts/main.js

```diff
@@ -25,4 +25,13 @@ function showPosts(entries) {
       entryClass.add("is--visible");
     }
   }
-}
\ No newline at end of file
+}
+
+document.getElementById("untappdConnect").addEventListener("click", untappdConnect);
+
+function untappdConnect() {
+  let CLIENTID = 'A0D5D7F766E859E3EF145BD051A3A576D2EA97CF'
```

Comment:
Is this a let or a const? For as far as I can see the variable is not going to change, as for the `REDIRECT_URL`. I'd do this instead
```suggestion
  const CLIENTID = 'A0D5D7F766E859E3EF145BD051A3A576D2EA97CF'
```